src/PVRIptvData.cpp

```diff
@@ -802,9 +810,9 @@ PVR_ERROR PVRIptvData::GetEPGForChannel(ADDON_HANDLE handle, const PVR_CHANNEL &
       tag.strPlotOutline      = myTag->strPlotOutline.c_str();
       tag.strPlot             = myTag->strPlot.c_str();
       tag.strOriginalTitle    = NULL;  /* not supported */
-      tag.strCast             = NULL;  /* not supported */
-      tag.strDirector         = NULL;  /* not supported */
-      tag.strWriter           = NULL;  /* not supported */
+      tag.strCast             = myTag->strActor.c_str();
```

Comment:
Do you have a sample of the XML used for this. Is this a single actor or a list of actors?

Reply:
This is one string with list of the actors separated with delimiter it's look nicer.
it's like category element, can be more that one, but used / readed only first, then better concatenate element value with delimiter

```xml
<programme channel="lietuvos-rytas-tv" start="20190815110500 +0300" stop="20190815121000 +0300">
<title>Šeimininkė</title>
<sub-title>1-as sezonas. 4-a serija</sub-title>
<desc>Alina – šeimininkė. Ji valdo visą šachtininkų miestelį...</desc>
<category>Movie / Drama</category>
<icon src="https://cdn.tvstart.com/img/programme/thumb/380708.jpg" />
<credits>
<director>Bata Nedich / Other Director</director>
<writer>Writer</writer>
<actor>Ekaterina Varchenko / Elena Stefanskaja / Sergej Radchenko</actor>
</credits>
</programme>
```

![screenshot007](https://user-images.githubusercontent.com/43538011/63081260-d8aa4280-bf4b-11e9-910d-cbb9abf1ddf0.png)
![screenshot006](https://user-images.githubusercontent.com/43538011/63081269-dc3dc980-bf4b-11e9-8d3b-48dd76c60dce.png)

Reply:
Ok, maybe also rename strActor as strCast. Similar to strEpisode name.

Reply:
I changed as strCast